Assets/MRTK/Core/Utilities/StandardShader/ProximityLight.cs

```diff
@@ -212,11 +215,6 @@ private void OnDrawGizmosSelected()
 
         private static void AddProximityLight(ProximityLight light)
         {
-            if (activeProximityLights.Count >= proximityLightCount)
```

Comment:
Are there any concerns with having the activeProximityLights list grow unboundedly? I guess we're currently allowing it to grow unboundedly, while only throwing a warning, but just want to double check

Reply:
Friendly ping on this @jonathoncobb

Reply:
Sorry letting this sit. I see the concern with this getting out of hand, though at a higher level, it's still bounded by the maximum number of pointers and thus light sources that will ever exist.